experiments/results/servers/etinydtls_psk_rwalk/sul.config

```diff
@@ -30,86 +36,53 @@
-        <RsaPrivateKey>
-            <modulus>26396090581505603361643260705843086668133841885251189234365835998792159294409530081912821732196528983039424482701894602969700780790114475786080489816864800570087377971442165131668631663930378088848021014527952725547303266244288170005142058645637046626203366799291519360532041490820845957941754130275124194303203071757346972395521319273455184768222972615636562395218812148990152883696459058876040710756735692007315440292418837658355824640242244343067991959558873311625321072693673194440794503401720278880490501301726411910417932510993430937104762261186098410906851635860351964996874234449577481641189385117140763431243</modulus>
-            <privateExponent>5401900710730627771890068397802271669331996999633626043476426940747981147391864404208535103410590150915128265895567548331928938949860618417732144123530077746097806023360579806001795747083849290135795929732042997925453276879753313946457196553935701502214620069763612274950878747499720554616702113237559938568903040381106063185004554593121518408318602168829805544647002321492266268869258473418784926741286804494954079637358685527248470924424392345822105293814397814252779449856425770962400499110382404768426074305201228933134536363414655411773881384148397564056432170192350103673212939320082584920472881578313574062561</privateExponent>
-        </RsaPrivateKey>
+        3059301306072a8648ce3d020106082a8648ce3d0301070342000436dfe2c6f9f2ed29da0a9a8f62684e916375ba10300c28c5e47cfbf25fa58f5271a0d4fcde1ab8785a3c786935a7cfabe93f987209daed0b4fabc36fc772f829
+		</certificateBytes>
+        <EcPublicKey>
+            <point>
+                <xFieldElementFp>
+                    <data>36DFE2C6F9F2ED29DA0A9A8F62684E916375BA10300C28C5E47CFBF25FA58F52</data>
+                    <modulus>FFFFFFFF00000001000000000000000000000000FFFFFFFFFFFFFFFFFFFFFFFF</modulus>
+                </xFieldElementFp>
+                <yFieldElementFp>
+                    <data>71A0D4FCDE1AB8785A3C786935A7CFABE93F987209DAED0B4FABC36FC772F829</data>
+                    <modulus>FFFFFFFF00000001000000000000000000000000FFFFFFFFFFFFFFFFFFFFFFFF</modulus>
+                </yFieldElementFp>
+            </point>
+            <group>SECP256R1</group>
+        </EcPublicKey>
+        <EcPrivateKey>
+            <privatekey>41C1CB6B51247A144321435B7A80E714896A33BBAD7294CA401455A194A949FA</privatekey>
+            <group>SECP256R1</group>
+        </EcPrivateKey>
+        <signatureGroup>SECP256R1</signatureGroup>
+        <publicKeyGroup>SECP256R1</publicKeyGroup>
     </defaultExplicitCertificateKeyPair>
 </Config>
-148990152883696459058876040710756735692007315440292418837658355824640242244343067991959558873311625321072693673194440794503401720278880490501301726411910417932510993430937104762261186098410906851635860351964996874234449577481641189385117140763431243</modulus>
-            <privateExponent>54019007107306277718900683978022716693319969996336260
\ No newline at end of file
+    <EcPublicKey>
```

Comment:
Something seems messed up below this line (and definitely in the last ones...)

Reply:
There is a (more general) bug when exporting the TLS-Attacker configuration. This affects all experiments in the results folder. I will change the sul.config, but the bug itself will need to be addressed as a separate issue.